tempus_fugit_minecraft/model.py

```diff
@@ -530,30 +530,37 @@ def collide(self, position: tuple, height: int) -> tuple:
                     break
         return tuple(p)
     
-    #issue 68
+    #Issue 68
     def handle_adjust_vision(self, dx, dy):
         self.player.adjust_sight(dx, dy)
 
-    #issue 68
+    #Issue 68
     def handle_change_active_block(self, index):
         self.player.select_active_item(index)
 
-    #issue 68
+    #Issue 68
     def handle_speed_change(self, increase):
         if increase:
-            self.player.speed_up()
+            self.player.increase_speed()
         else:
-            self.player.speed_down()
+            self.player.decrease_speed()
 
-    #issue 68
+     #Issue 39
+    def handle_jump_change(self, increase):
+        if increase:
+            self.player.increase_jump()
+        else:
+            self.player.decrease_jump()
```

Comment:
Needs to change to be more specific. With how it is, if increase is anything that's not False or 0, it will always default to the first if statement.

Reply:
Return None, updated! Thank you!